app/templates/show-project.hbs

```diff
@@ -54,7 +54,7 @@
               <span class="label dark-gray">
                 {{~model.dcp_ulurp_nonulurp~}}
                 <sup>
-                  {{~icon-tooltip tip='Uniform Land Use Review Procedure (ULURP) is a procedure whereby applications affecting the land use of the city are publicly reviewed within mandated time frames. A project is designated ULURP if a minimum of one action (e.g. a Zoning Map Amendment) within the application process is designated ULURP. Key participants in the process are the Department of City Planning (DCP), the City Planning Commission (CPC), Community Boards, the Borough Presidents, the Borough Boards, the City Council and the Mayor.'~}}
+                  {{icon-tooltip tip='Uniform Land Use Review Procedure (ULURP) is a procedure whereby applications affecting the land use of the city are publicly reviewed within mandated time frames. Key participants in the process are the Department of City Planning (DCP), the City Planning Commission (CPC), Community Boards, the Borough Presidents, the Borough Boards, the City Council and the Mayor.'}}
```

Comment:
We keep the tilde `{{~` to remove padded newlines (so we can organize into multiline).